lib/private/DB/PostgreSqlMigrator.php

```diff
@@ -42,7 +42,7 @@ protected function getDiff(Schema $targetSchema, \Doctrine\DBAL\Connection $conn
 					}
 					$fromDefault = $column->fromColumn->getDefault();
 					$toDefault = $column->column->getDefault();
-					$fromDefault = trim($fromDefault, "()");
+					$fromDefault = trim((string) $fromDefault, "()");
```

Comment:
```suggestion
					$fromDefault = trim($fromDefault ?? "", "()");
```

Reply:
I'm actually not sure if `?? ''` is going to work. it might solve `null` but not numeric values in case doctrine ever goes with that, so casting to string feels saver.

if we are touching it again anyway lets use single quotes:
```suggestion
					$fromDefault = trim((string) $fromDefault, '()');
```

Reply:
Since trim() expects a string, we should make sure, it is safely converted to a string, even if it is null, boolean, numeric or what ever. I second @nickvergessen, feels safer with the cast.

... changed the quotes, btw.

Reply:
Sounds good to me if all those value ranges matter.